kind/Dockerfile

```diff
@@ -18,4 +18,4 @@ WORKDIR kind
 COPY chart chart/
 COPY helmfile.yaml kind.gotmpl kind.yaml start-kind.sh ./
 
-CMD "./start-kind.sh"
\ No newline at end of file
+CMD "./kind/start-kind.sh"
```

Comment:
I don't think this is going to work because start-kind.sh is copied into the container, but not in the `kind` path.

Reply:
everything is copied as-is,  and that is the location it is found from CWD in the container